Lib/test/test_syntax.py

```diff
@@ -835,6 +835,39 @@
      ...
    SyntaxError: invalid syntax
 
+Check that an exception group with missing parentheses
```

Comment:
This terminology conflicts with PEP 654 exception groups

cc @iritkatriel

Reply:
Thanks for pointing that out. If exceptions group pep is merged we will surely change this sentence.

On the other hand, do you have some alternative phasing for the error you think would be clearer?

Reply:
isinstance calls this thing a classinfo

Reply:
https://docs.python.org/3.10/library/functions.html#isinstance

Reply:
Unfortunately I find that even more confusing :(

Reply:
exception sequence?

Reply:
"multiple exception types"?

Reply:
I started a mailing list thread in python-dev@python.org "a name for the ExceptHandler.type when it is a literal tuple of types"

Reply:
@iritkatriel  yeah I like it: `SyntaxError: multiple exception types must be parenthesized`

Reply:
Opened https://github.com/python/cpython/pull/25996